File: Loop10.scala
//10)Check if a number is prime using a "while" loop.
object Loop10 {
  def main(args:Array[String]):Unit={
    var num=7
    var count=0
    for (i<- 1 to num){
      if (num%i==0){
        count=count+1

      }
    }
    if(count==2){
      print("Given Number is prime: "+num)
    }
  }

}

File: Loop10.py
#10)Check if a number is prime using a "while" loop.

num =11
count=0
for i in range(1,num+1):
    if(num%i==0):
        count=count+1

if(count==2):
    print("GIven number is prime")
else:
    print("Give number is not a prime ")